RealmModelGenerator/Entity.swift

```diff
@@ -97,6 +98,18 @@ class Entity {
         return relationship
     }
     
+    func createRelationship(relationship: Relationship) -> Void {
```

Comment:
Why did you add a function to add a relationship that is constructed out side of the entity class?
The above create relationship should handle all of the relationship building needs.
Why would you like to have the Relationship constructed and added to the Entity as opposed to the Entity create the relationship internally?